set.mm

```diff
@@ -3860,12 +3860,9 @@ Here we define disjunction (logical 'or') ` \/ ` ( ~ df-or ) and conjunction
 
   $( Declare connectives for disjunction ('or') and conjunction ('and'). $)
   $c \/ $. $( Vee (read:  'or') $)
-  $c /\ $. $( Wedge (read:  'and') $)
```

Comment:
I thought we agreed that conjunction was to be introduced before disjunction ?

Reply:
I aim at being not so disruptive, so people can still find most of the stuff most of the time.